src/renderer/components/DownloadsDropdown/Update/UpdateWrapper.tsx

```diff
@@ -11,14 +11,25 @@ import 'renderer/css/components/Button.scss';
 
 type OwnProps = {
     item: DownloadedItem;
+    appName: string;
 }
 
-const UpdateWrapper = ({item}: OwnProps) => {
+const UpdateWrapper = ({item, appName}: OwnProps) => {
     if (item.state === 'available') {
-        return <UpdateAvailable item={item}/>;
+        return (
+            <UpdateAvailable
+                item={item}
+                appName={appName}
+            />
+        );
     }
     if (item.state === 'completed') {
-        return <UpdateDownloaded item={item}/>;
+        return (
+            <UpdateDownloaded
+                item={item}
+                appName={appName}
+            />
+        );
```

Comment:
Not introduced in this PR but wondering why are these two cases separate? Seems they return the same thing

Reply:
One is `UpdateAvailable` the other is `UpdateDownloaded`, there are separate components.